README.md

```diff
@@ -32,6 +36,60 @@ const rule = {
 };
 ```
 
+## With Expo
+When using Expo, you will need to add a webpack file that will be used for web builds in order to add the above code.
+The following command will create the file in the way expected by Expo and will also install the **@expo/webpack-config** dependency which is necessary.
+```$
+npx expo customize webpack.config.js
+```
+
+If you haven't set up Expo to export web projects at all, you will need to install the following dependencies.
+```
+npx expo install react-native-web react-dom
+```
+
+Below is an example implementation of **webpack.config.js** in an expo project.
+```js
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+
+
+///////
+// NOTE: Webpack is only used for bundling web
+///////
+
+
+module.exports = async function (env, argv) {  
+  const config = await createExpoWebpackConfigAsync(env, argv);
+
+  config.resolve.alias = {
+    ...config.resolve.alias,
+
+    // Set up aliases so the web versions are used (Rather than changing imports in each file)
+    'react-native': 'react-native-web',
+    'react-native-webview': 'react-native-web-webview',
+
+  };
+
+  // Prevents an error the react-native-web-webview recieves when building
```

Comment:
Why this rule is needed isn't clearly explained. I don't understand it well myself so I've been vague in my comment here, but someone should clarify if possible.